src/character.cpp

```diff
@@ -10948,15 +10944,16 @@ void Character::process_effects()
                         add_effect( effect_cramped_space, 2_turns, true );
                     }
                     is_cramped_space = true;
+                    return;
```

Comment:
These early returns were the cause of #70389 before, which was fixed in #70389 . In this change, the `return`-statements here and on line 10956 are added back. Do you know if this change accidentally re-introduces #70389 again now if you sleep in a cramped space in a vehicle?

Reply:
I don't, and they probably do. I didn't realize I was re-introducing the bug.